### PR TITLE
Delegated admin v2

### PR DESCRIPTION
**Pull request by sandrinodimattia**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/extensions/delegated-admin.md [hunk 1]
```diff
@@ -130,44 +357,23 @@ There are two available views, *Users* and *Logs*. At the *Users* view you can s
...
 Navigate to the [Extensions](${manage_url}/#/extensions) page and go to the settings of the **Delegated Administration** extension. We are going to set a custom title and a custom css file:
-- Set the **Title** to `Fabrikam User Management`.
-- Set the **Custom_CSS** to `https://rawgit.com/auth0-extensions/auth0-delegated-administration-extension/master/docs/theme/fabrikam.css`.
+
+- Set the **TITLE** to `Fabrikam User Management`.
+- Set the **CUSTOM_CSS** to `https://rawgit.com/auth0-extensions/auth0-delegated-administration-extension/master/docs/theme/fabrikam.css`.
 
 Save your changes, navigate to the extension and login. The dashboard now looks like that:
 
 ![](/media/articles/extensions/delegated-admin/custom-dashboard.png)
```
Comment (sandrinodm) [inline articles/extensions/delegated-admin.md:420]: This screenshot needs to be updated.

File: articles/extensions/delegated-admin.md [hunk 2]
```diff
@@ -62,20 +93,216 @@ Set the following configuration variables:
...
 
-Notice that there is no Sign Up option. That's because we disabled it earlier. 
+Notice that there is no Sign Up option. That's because we disabled it earlier.
 
 Once you provide valid credentials you are navigated to the *Delegated Administration Dashboard*.
 
 ![](/media/articles/extensions/delegated-admin/standard-dashboard.png)
 
-There are two available views, *Users* and *Logs*. At the *Users* view you can see all the users displayed and perform certain actions on them. In the table below you can see all the options you can perform on a user, which ones are available via the [dashboard](${manage_url}/#/) and which via the extension. 
+### Configure Hooks
```
Comment (sandrinodm) [inline articles/extensions/delegated-admin.md:114]: We need a screenshot here.

File: articles/extensions/delegated-admin.md [hunk 3]
```diff
@@ -62,20 +93,216 @@ Set the following configuration variables:
...
+To access your newly created dashboard, navigate to *[Extensions](${manage_url}/#/extensions) > Installed Extensions* and click on the **Delegated Administration Dashboard**. A new tab will open and display the login prompt.
 
 ![](/media/articles/extensions/delegated-admin/login-prompt.png)
 
-Notice that there is no Sign Up option. That's because we disabled it earlier. 
+Notice that there is no Sign Up option. That's because we disabled it earlier.
 
 Once you provide valid credentials you are navigated to the *Delegated Administration Dashboard*.
 
 ![](/media/articles/extensions/delegated-admin/standard-dashboard.png)
```
Comment (sandrinodm) [inline articles/extensions/delegated-admin.md:112]: This screenshot needs to be updated.

File: articles/extensions/delegated-admin.md [hunk 4]
```diff
@@ -2,58 +2,89 @@
...
+}
+```
+
+## Install the Extension
+
 We are now ready to setup our new extension. Before we do so head back to your new Client and copy the **Client ID** value.
 
 To install and configure this extension, click on the **Delegated Administration** box in the list of provided extensions on the [Extensions](${manage_url}/#/extensions) page of the dashboard. The **Install Extension** window will open.
 
 ![](/media/articles/extensions/delegated-admin/install-extension.png)
```
Comment (sandrinodm) [inline articles/extensions/delegated-admin.md:86]: This screenshot needs to be updated.

File: articles/extensions/delegated-admin.md (hunk 2 above)
Comment (mpaktiti) [reply articles/extensions/delegated-admin.md:114]: Done.

File: articles/extensions/delegated-admin.md (hunk 3 above)
Comment (mpaktiti) [reply articles/extensions/delegated-admin.md:112]: Done (updated with sandrino user)

File: articles/extensions/delegated-admin.md (hunk 4 above)
Comment (mpaktiti) [reply articles/extensions/delegated-admin.md:86]: I didn't update this one, not sure if I should. I can see the same thing, only the version at the top right changes.

File: articles/extensions/delegated-admin.md (hunk 1 above)
Comment (mpaktiti) [reply articles/extensions/delegated-admin.md:420]: Done (with kelly user). Not sure if this is correct, it looked custom, while sandrino user has the auth0 ui.

**Pull request merged**